testing/tests/DevExpress.ui.widgets/overlay.tests.js

```diff
@@ -2032,26 +2050,10 @@ testModule('close on target scroll', moduleConfig, () => {
         assert.strictEqual(overlay.option('visible'), true, 'overlay should not be hidden if show animation is not completed');
     });
 
-    test('overlay should be hidden if any of jQuery Event target\'s parents were scrolled', function(assert) {
```

Comment:
Why do we delete this test?

Reply:
I think, this test looks like 'funcional', not looks like 'utin-test'. My goal in the next: 'do not have tests on no existing code'. Our exists code is cover by other unit-tests and his test looks like 'extra'.

Reply:
I returned it back